src/cueUtilities.js

```diff
@@ -129,6 +129,32 @@ module.exports = function (params, cy, api) {
             x : x,
             y : y
           };
+        } else if (options().expandCollapseCuePosition === 'top-right') {
+          var offset = 1;
+          var size = cy.zoom() < 1 ? rectSize / (2*cy.zoom()) : rectSize / 2;
+
+          var x = node.position('x') + node.width() / 2 + parseFloat(node.css('padding-left'))
+                  - parseFloat(node.css('border-width')) - size - offset;
+          var y = node.position('y') - node.height() / 2 - parseFloat(node.css('padding-top'))
+                  + parseFloat(node.css('border-width')) + size + offset;
+
+          cueCenter = {
+            x : x,
+            y : y
+          };
+        } else if (options().expandCollapseCuePosition === 'top-center') {
+          const x = node.position('x');
+          let y;
+          if (cy.expandCollapse('get').isExpandable(node)) {
+              y = node.position('y');
```

Comment:
I think the above code was intended to center the node vertically, which doesn't apply for 'top-center', so please remove.